thunder/executors/nvfuserex_impl.py

```diff
@@ -1088,6 +1092,7 @@ def convert_element_type(
 
 
 register_supported(PrimIDs.CONVERT_ELEMENT_TYPE, convert_element_type, _convert_element_type_check)
+register_supported(dtensor_convert_element_type_prim, convert_element_type, _convert_element_type_check)
```

Comment:
If for the nvFuser executor `PrimIDs.CONVERT_ELEMENT_TYPE` and `dtensor_convert_element_type_prim` are the same, how are they different from each other in Thunder?